CONTRIBUTING.md

```diff
@@ -109,6 +109,10 @@ single user, or just a few co-located users).
 
 The `#[cfg(test)] mod tests {}` module goes on the very bottom, if present.
 
+When defining new module definitions with `mod` they should be placed after
```

Comment:
I think the way I usually do it is a little different, which is to keep private `use`s together with public `use`s (since I think it's confusing to separate imports from the same source in different locations). For imports from a locally-defined module, those should be grouped with the module (whether declaration or definition).

Reply:
> For imports from a locally-defined module, those should be grouped with the module (whether declaration or definition).

Yeah, agree. Here's an example of that happening: https://github.com/rustls/rustls/blob/main/rustls/src/conn.rs#L351-L352

Reply:
Do either of you want to suggest an edit? I'm not sure the best way to translate this guidance into text.

Reply:
Figured this might be easiest as a separate PR: https://github.com/rustls/rustls/pull/2271.